Controller/CRUDController.php

```diff
@@ -234,7 +234,8 @@ public function editAction($id = null)
         // the key used to lookup the template
         $templateKey = 'edit';
 
-        $id = $id ?: $request->get($this->admin->getIdParameter());
+        // If the $id variable is provided AND the child ID is not needed, do not reload it.
```

Comment:
> Can be done indeed: https://3v4l.org/1tlZW

My concern was not inheritance, but having Symfony match `$id` with the corresponding request parameter.